tests/9-scalable_proc/CMakeLists.txt

```diff
@@ -97,6 +98,7 @@ scalable_proc(
   TOP_FILE basys3_top.v
   BOARD basys3
   INPUT_IO_FILE basys3.pcf
+  INPUT_SDC_FILE ${symbiflow-arch-defs_SOURCE_DIR}/xilinx/xc7/tests/common/basys3.sdc
```

Comment:
why do we need absolute path here? Other files use relative paths

Reply:
This because the SDC file is not in the scalable proc tests, but I think it would actually be better to copy the basys3.sdc from the xc7 path. Because these set of tests should be detached from the particular architecture.

Reply:
Done